command/server/listener.go

```diff
@@ -124,3 +140,36 @@ func (cg *certificateGetter) getCertificate(clientHello *tls.ClientHelloInfo) (*
 
 	return cg.cert, nil
 }
+
+func parseCiphers(cipherStr string) ([]uint16, error) {
+	suites := []uint16{}
+	ciphers := strings.Split(cipherStr, ":")
```

Comment:
Can we use the inbuilt helper `strutil.ParseDedupAndSortStrings()` here? Also, it might be better to delimit the entries with a `,` instead of a `:` just to be consistent with several API field delimiters.